Validation.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/nyaruka/phonenumbers"
+)
+
+// ---
+// regexes
+// ---
+var (
+	// implemented to https://nl.wikipedia.org/wiki/Postcode#Postcodes_in_Nederland
+	//
+	// go-staticcheck is angry here, but this is PCRE-compliant
+	PostalCodeRegex *regexp.Regexp = regexp.MustCompile(`^[1-9]\d{3}\w{2}$`)
+	// this does not need to be fancy, just needs to check if it is somewhat valid.
+	//
+	// checks for `*@*.*`, which... complies with _some_ RFC at least... right...?
+	EmailRegex *regexp.Regexp = regexp.MustCompile(`[\w\d]+@[\w\d]+[.][\w]+`) // jesus christ this is a terrible regex
+)
+
+// ---
+// validation functions
+// ---
+
+// this function takes a postal code and throws a regex at it to see if it is a dutch postal code
+//
+// it is possible to first check if it is just 4 numbers, making it belgian, but that might be beyond the scope
+func validatePostalCode(code string) error {
+	if PostalCodeRegex.FindString(code) == "" {
+		return errors.New("Postcode is onjuist, probeer het zo: 4818 AJ.")
```

Comment:
`go fmt` heeft liever geen interpunctie en hoofdletters in error messages

nou is het niet erg hier want de error wordt direct naar json omgetoverd, maar ik krijg er wel gele kringeltjes van in mn IDE